components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -1554,6 +1554,28 @@ export class GitpodServerEEImpl extends GitpodServerImpl<GitpodClient, GitpodSer
         return prebuild;
     }
 
+    async cancelPrebuild(projectId: string, prebuildId: string): Promise<void> {
```

Comment:
in theory we could get the `projectId` from the workspace of that prebuild. 
what's the advantage of adding it to the list of args, besides remove a couple of db queries?

Reply:
> what's the advantage of adding it to the list of args, besides remove a couple of db queries?

None. Removing a few unnecessary DB queries sounds great, and this is information we typically already have when making the call, so we might as well not omit it. 🙂 Do you see it otherwise?